src/main/java/com/alkemy/ong/controller/v1/UserController.java

```diff
@@ -1,16 +1,38 @@
 package com.alkemy.ong.controller.v1;
 
+import com.alkemy.ong.dto.UserDTO;
+import com.alkemy.ong.dto.UserResponseDTO;
 import com.alkemy.ong.service.UserService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-import static com.alkemy.ong.controller.ControllerConstants.V_1_USERS;
+
+import javax.validation.Valid;
+
+
+import static com.alkemy.ong.controller.ControllerConstants.REQ_MAPP_AUTH;
+import static com.alkemy.ong.controller.ControllerConstants.REQ_MAPP_REGISTER;
 
 @RestController
-@RequestMapping(V_1_USERS)
```

Comment:
Please do not change this Controller and leave the URI as /v1/users for all operations in users, use a Controller for authorization instead.